samples/nrf9160/fmfu_smp_svr/README.rst

```diff
@@ -42,7 +42,12 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
-To use the UART overlay for increasing the transfer speed, add the ``-DOVERLAY_FILE=uart.overlay`` flag to your build.
+To use the UART overlay for increasing the transfer speed, add the ``-DDTC_OVERLAY_FILE=uart.overlay`` flag to your build.
+The following command builds the sample with the UART overlay for nRF9160 DK:
```

Comment:
We focus on SES in the instructions, so we should add a link for how to add this option.

```suggestion
To use the UART overlay for increasing the transfer speed, add the ``-DDTC_OVERLAY_FILE=uart.overlay`` flag to your build.
See :ref:`cmake_options` for instructions on how to add this flag.
For example, when building on the command line, add the UART overlay for nRF9160 DK in the following way:
```

Reply:
Thanks I have added this now. Previously, I had worded it similarly to how it is worded in the mqtt_simple example [here](https://github.com/nrfconnect/sdk-nrf/blob/master/samples/nrf9160/mqtt_simple/README.rst#configuration-files). Perhaps we need to change that to include the SES instructions as well?

Reply:
Agree - I'll make a task.